metabrainz/templates/index/funding.json

```diff
@@ -0,0 +1,233 @@
+            "webpageUrl": {
+                "url": "https://listenbrainz.org",
+                "wellKnown": "https://listenbrainz.org/.well-known/funding-manifest-urls" 
+            },
+            "repositoryUrl": {
+                "url": "https://github.com/metabrainz/listenbrainz-server",
+                "wellKnown": "https://github.com/metabrainz/listenbrainz-server/blob/main/.well-known/funding-manifest-urls"
+            },
+            "licenses": ["GPL-2.0-or-later", "CC-BY-NC-SA-3.0", "CC0-1.0"],
+            "tags": ["music-community", "recommendations", "fresh-releases", "listening-data", "open-data", "data-service-provider", "non-profit"]
+        },
+        {
+            "guid": "bookbrainz",
+            "name": "BookBrainz",
+            "description": "BookBrainz is an ambitious open-source project dedicated to creating a comprehensive online database of every published book, magazine, journal, and other publication ever written. As part of the MetaBrainz Foundation, alongside MusicBrainz and other projects, BookBrainz empowers users to contribute and edit information about books, including titles, authors, publication dates, ISBNs, and more. This collaborative effort fosters a vast and accurate resource for book lovers, researchers, and libraries worldwide. By making this data freely accessible, BookBrainz promotes knowledge sharing and fosters a deeper understanding of the literary world.",
+            "webpageUrl": {
+                "url": "https://bookbrainz.org",
+                "wellKnown": "https://bookbrainz.org/.well-known/funding-manifest-urls" 
+            },
+            "repositoryUrl": {
+                "url": "https://github.com/metabrainz/bookbrainz-site",
+                "wellKnown": "https://github.com/metabrainz/bookbrainz-site/blob/main/.well-known/funding-manifest-urls"
+            },
+            "licenses": ["GPL-2.0-or-later", "CC-BY-NC-SA-3.0", "CC0-1.0"],
+            "tags": ["book-community", "book-metadata", "open-data", "isbn", "data-service-provider", "non-profit"]
+        }
+
+    ],
+
+    "funding": {
+        "channels": [
+            {
+                "guid": "paypal",
+                "type": "payment-provider",
+                "address": "paypal@metabrainz.org",
+                "description": "Make a donation to MetaBrainz via PayPal."
+            },
+            {
+                "guid": "bank",
+                "type": "bank",
+                "address": "IBAN: BE96 9670 3472 0205\nBIC: TRWIBEB1XXX",
+                "description": "Make a donation to MetaBrainz via SEPA bank transfer."
+            },
+            {
+                "guid": "credit-card",
+                "type": "payment-provider",
+                "address": "https://metabrainz.org/donate",
+                "description": "Make a donation to MetaBrainz via a Credit Card payment."
+            }
```

Comment:
Should we also add the Github donations thing?
https://github.com/sponsors/metabrainz

And stripe?